I have developed a full-fledged REST API for managing contacts using …

Travellin91 opened the pull request:
…Node.js, Express, and MongoDB. Your API allows you to:

I have developed a full-fledged REST API for managing contacts using Node.js, Express, and MongoDB. Your API allows you to:

Retrieve contact data.
Create new contacts.
Update contact information and their "favorite" flag. Delete contacts.
I set up a connection to the MongoDB Atlas database, utilizing confidential data from the .env file. Your API runs on a server, listens for incoming requests, and provides responses in JSON format. The code is organized into models, controllers, and routes for better structuring.

Travellin91 committed:
I have developed a full-fledged REST API for managing contacts using Node.js, Express, and MongoDB. Your API allows you to:

I have developed a full-fledged REST API for managing contacts using Node.js, Express, and MongoDB. Your API allows you to:

Retrieve contact data.
Create new contacts.
Update contact information and their "favorite" flag.
Delete contacts.
I set up a connection to the MongoDB Atlas database, utilizing confidential data from the .env file. Your API runs on a server, listens for incoming requests, and provides responses in JSON format. The code is organized into models, controllers, and routes for better structuring.

ceREAL133 commented on app.js:
```diff
@@ -1,25 +1,42 @@
+
+app.use(cors());
+app.use(express.json());
+
```
This is sensitive info, hide it in .env file

Travellin91 committed:
fixed

I have hidden sensitive information, such as database connection details, in the .env file. This ensures security and prevents potential leakage of confidential data, as these details are no longer hardcoded in the code.

ceREAL133 commented on app.js:
```diff
@@ -1,25 +1,43 @@
-app.use(logger(formatsLogger))
-app.use(cors())
-app.use(express.json())
+const uriDb = `mongodb+srv://${process.env.DB_USERNAME}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}/db-contacts?retryWrites=true&w=majority`;
```
You could hide all the uri string
